src/ploomber/sources/notebooksource.py

```diff
@@ -266,11 +268,19 @@ def __init__(self,
         if self._path is not None and ext_in is None:
             self._ext_in = self._path.suffix[1:]
         elif self._path is None and ext_in is None:
-            raise ValueError('"ext_in" cannot be None if the notebook is '
-                             'initialized from a string. Either pass '
-                             'a pathlib.Path object with the notebook file '
-                             'location or pass the source code as string '
-                             'and include the "ext_in" parameter')
+
+            if Path(self._primitive).exists():
+                raise ValueError(
+                    '{} looks like a path to a file. '
+                    'Perhaps you meant passing a pathlib.Path object'.format(
```

Comment:
please show a code example as part of the error message:

```
perhaps you meant passing a pathlib.Path object like this:

from pathlib import Path
Path('value-that-the-user-passed')
```

Reply:
> please show a code example as part of the error message:

sorry, I dont get the question.

Reply:
to be more user-friendly, modify the error message so we tell the user how to fix their code (otherwise they need to figure out that they need to add an import and then enclose their string using `Path`. it'll make it easier for them to know how to fix it :)

Reply:
Oh I see your point. make sense to me.

Let me update the error message in the next revision

Reply:
thanks!